modules/ml/include/opencv2/ml.hpp

```diff
@@ -691,6 +699,40 @@ class CV_EXPORTS_W SVM : public StatModel
                     ParamGrid degreeGrid = getDefaultGrid(DEGREE),
                     bool balanced=false) = 0;
 
+    /** @brief Trains an %SVM with optimal parameters
```

Comment:
If you will use `Ptr<TrainData>` to pass train data, all the below description can be replaced with the `/** @overload */` command.